criprobe/tests/test_cri.py

```diff
@@ -11,6 +12,21 @@ def test_init(self):
         self.assertEqual(p.probes[0], {'Type': 'CR-100'})
         self.assertEqual(p.probes[1], {'Type': 'CR-250'})
 
+    def test_init_real_probe(self):
+        # this test will only be meaningful if real probes are connected
+        # if no probes are connected it will simply pass
+        p = cri.CriProbe()
+
+        # Only check results if there are real probes attached
+        if p.probes:
+            # check to make sure each probe in the list has valid info
+            for probe in p.probes:
```

Comment:
Not sure what happened to the comment I tried to leave here.  I think you should include tests for model and instrument type here